CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixed
 - Filters: keep value if typeof match on operator change
+- Pagination: always display pagination for multiple pages dataset
```

Comment:
In weaverbird, we generally avoid doing merges and prefer to rebase to keep a linear git history.
If you don't have to do this it's not a big deal, we can also squash and merge this PR (which will hide the merge) as long as the PR stays small.

Reply:
Good to know, i will take care for future conflicts :+1: